### PR TITLE
[opentitanlib] fix English Breakfast build

### PR DESCRIPTION
**Pull request by nbdd0121**
Fix English Breakfast build after #21682.

This is a quite hacky approach, but I don't want to implement proper multitop to fix the issue...

### CODE REVIEW

**Review by jwnrt — APPROVED** [review]
This looks okay to me given the other things that the englishbreakfast generation script does. How well are we testing englishbreakfast in CI?

**Comment by pamaury** [comment]
English breakfast is a hack anyway so that approach seems fine with me.

**Review by pamaury — APPROVED** [review]

File: sw/host/opentitanlib/bindgen/BUILD [hunk 1]
```diff
@@ -7,41 +7,56 @@ load("@rules_rust//bindgen:bindgen.bzl", "rust_bindgen_library")
 
 package(default_visibility = ["//visibility:public"])
 
+_TOPLEVEL = "earlgrey"
```
[the remark this answers is not in the document]
Comment (nbdd0121) [reply sw/host/opentitanlib/bindgen/BUILD:11]: Done

**Review by a-will — APPROVED** [review]
This seems fine.

I think I'd have preferred to use a bazel select() with some config instead of textual replacement from prepare_sw.py, but hopefully that whole script won't be around _too_ much longer. So I won't quibble if we worry about improving the form later, when we get to multi-top support, hehe.

**Comment by pamaury** [comment]
> This seems fine.
> 
> I think I'd have preferred to use a bazel select() with some config instead of textual replacement from prepare_sw.py, but hopefully that whole script won't be around _too_ much longer. So I won't quibble if we worry about improving the form later, when we get to multi-top support, hehe.

As far as I know, we do not even have a bazel switch for english breakfast (we only have a switch to disable bitmanip), we just patch files. This PR just continues with the tradiction ;)

**Pull request merged**